### DISCUSSION TITLE
Keeps asking me to buy a other lifetime key

### DISCUSSION THREAD

**Discussion by maxlawz** [aclap-dev/video-downloadhelper#1389] [Help and Q&A]
### Your question

I have been using this app for a couple years now, and throughout those years I have purchased a license twice. The second time I bought the license, I was confused, and I thought maybe it was only for like a year, so the second time I bought it, I paid for the lifetime license. Now I'm getting the same issue; it's asking me to buy another license. I noticed the application has updated, and it no longer has my older license information.

### Addon & CoApp version and details

Version 9.1.1.0 (stable), target google, locale en-US.

**Comment by jcfrog** [comment]
Hi, you probably just need to revalidate your license key: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key
